WTWSMS/common/religions/00_religions.txt

```diff
@@ -1336,6 +1353,7 @@ muslim = {
 		uses_jizya_tax = no
 		can_hold_temples = no
 		defensive_attrition = yes #Yazidi originated deep in the mountains and hostile terrain
+		independence_war_score_bonus = 75

```

Comment:
War score isn't what affected it, but the battles themselves are directly impacted by it.